packages/reports/tests/acceptance/metric-parameters-test.js

```diff
@@ -90,65 +90,6 @@ module('Acceptance | navi-report - metric parameters', function(hooks) {
     config.navi.FEATURES.enableRequestPreview = originalFeatureFlag;
   });
 
-  test('adding and removing metrics - enableRequestPreview', async function(assert) {
-    assert.expect(7);
-
-    const originalFeatureFlag = config.navi.FEATURES.enableRequestPreview;
-
-    config.navi.FEATURES.enableRequestPreview = true;
-
-    await visit('/reports/1/view');
-
-    //adding a metric with default params
-    await clickItem('metric', 'Platform Revenue');
```

Comment:
do we have another test for default params?

Reply:
Yup it's in this test file.

Reply:
I mean, should we still test the default param case when the flag is turned on, even if there's no config that opens up?